src/lumigo_opentelemetry/instrumentations/__init__.py

```diff
@@ -12,10 +13,24 @@ class AbstractInstrumentor(ABC):
     def __init__(self, instrumentation_id: str):
         self._instrumentation_id = instrumentation_id
 
+    def is_applicable(self) -> bool:
+        tracing_enabled = (
+            os.environ.get("LUMIGO_ENABLE_TRACES", "TRUE").upper() == "TRUE"
+        )
```

Comment:
why lowercase in `src/lumigo_opentelemetry/__init__.py` (`tracing_enabled = os.getenv("LUMIGO_ENABLE_TRACES", "true").lower() == "true"`) but uppercase here?

Reply:
No good reason, just an oversight :) changed